tools/travis_mvn_watchdog.sh

```diff
@@ -543,35 +543,45 @@ case $TEST in
 			printf "Running end-to-end tests\n"
 			printf "==============================================================================\n"
 
-			printf "\n==============================================================================\n"
-			printf "Running Wordcount end-to-end test\n"
-			printf "==============================================================================\n"
-			FLINK_DIR=build-target CLUSTER_MODE=cluster test-infra/end-to-end-test/test_batch_wordcount.sh
-			EXIT_CODE=$(($EXIT_CODE+$?))
-
-			printf "\n==============================================================================\n"
-			printf "Running Kafka end-to-end test\n"
-			printf "==============================================================================\n"
-			FLINK_DIR=build-target CLUSTER_MODE=cluster test-infra/end-to-end-test/test_streaming_kafka010.sh
-			EXIT_CODE=$(($EXIT_CODE+$?))
-
-			printf "\n==============================================================================\n"
-			printf "Running class loading end-to-end test\n"
-			printf "==============================================================================\n"
-			FLINK_DIR=build-target CLUSTER_MODE=cluster test-infra/end-to-end-test/test_streaming_classloader.sh
-			EXIT_CODE=$(($EXIT_CODE+$?))
-
-			printf "\n==============================================================================\n"
-			printf "Running Shaded Hadoop S3A end-to-end test\n"
-			printf "==============================================================================\n"
-			FLINK_DIR=build-target CLUSTER_MODE=cluster test-infra/end-to-end-test/test_shaded_hadoop_s3a.sh
-			EXIT_CODE=$(($EXIT_CODE+$?))
-
-			printf "\n==============================================================================\n"
-			printf "Running Shaded Presto S3 end-to-end test\n"
-			printf "==============================================================================\n"
-			FLINK_DIR=build-target CLUSTER_MODE=cluster test-infra/end-to-end-test/test_shaded_presto_s3.sh
-			EXIT_CODE=$(($EXIT_CODE+$?))
+			if [ $EXIT_CODE == 0 ]; then
+				printf "\n==============================================================================\n"
+				printf "Running Wordcount end-to-end test\n"
+				printf "==============================================================================\n"
+				FLINK_DIR=build-target CLUSTER_MODE=cluster test-infra/end-to-end-test/test_batch_wordcount.sh
+				EXIT_CODE=$(($EXIT_CODE+$?))
```

Comment:
If `$EXIT_CODE == 0` then why not simply set `EXIT_CODE=$?`?

Reply:
jup we can simplify things now.